### PR TITLE
uploaded docker development blog post

### PR DESCRIPTION
**Pull request by kawaii**
This is in relation to #7.

### CODE REVIEW

File: november-2017/docker.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+With the recent announcements of the 1.10 branch we've also got some great news for both plugin developers and theme designers. In recent weeks, work has been underway to create an official Docker image for the MyBB 1.8 series. This work has been taken on by a community member who proposed such a solution and set to work on getting it done. Also provided within the repository are two example docker-compose.yml files, to be used with Docker Compose or Swarm. Docker is a technology for creating lightweight containers to manage and orchestrate software. It works across multiple platforms, and makes it easy to quickly get a live instance of the current MyBB version running with a variety of different configurations without requiring much effort to install database management systems and such on your machine. Docker is a brilliant tool for developers to quickly prototype ideas in a fresh, consistent environment. You can find out more information on Docker [here](https://docs.docker.com/get-started/).
+
+Developers can immediately download and start using the current MyBB 1.8.13 Docker image from our official Docker Hub registry account by running the following command:
```
Comment (euantorano) [inline november-2017/docker.md]: Maybe a good idea to link to the Docker Hub repository: https://hub.docker.com/r/mybb/mybb/

File: november-2017/docker.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+```
+docker pull mybb/mybb:latest
+```
+Alternatively, for those who would like to view the source, make ammendments or just generally prefer to build their own images - you can view the official GitHub repository [here](https://github.com/mybb/docker-compose/). Ample instructions for use are provided within the [README.md](https://github.com/mybb/docker-compose/blob/master/README.md) file. This project is far from complete and we do not recommend it for production use, only for development and staging systems. MyBB's current installation and upgrade systems make full automation with Docker a difficult feat to acomplish but we're hoping to include easier ways to do this in the future 1.10 series of releases.
+
+We hope that avid plugin and theme developers make good use of the Docker images that we'll be releasing and improving upon, any and all contructive feedback (and pull requests!) is welcome.
+
+Thanks,
+the MyBB Team
```
Comment (euantorano) [inline november-2017/docker.md]: `The MyBB Team`

**Review by euantorano — CHANGES_REQUESTED** [review]

**Review by euantorano — APPROVED** [review]

File: november-2017/docker.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Developers can immediately download and start using the current MyBB 1.8.13 Docker image from our official Docker Hub registry account (found [here](https://hub.docker.com/r/mybb/mybb/)) by running the following command:
+```
+docker pull mybb/mybb:latest
+```
+Alternatively, for those who would like to view the source, make ammendments or just generally prefer to build their own images - you can view the official GitHub repository [here](https://github.com/mybb/docker-compose/). Ample instructions for use are provided within the [README.md](https://github.com/mybb/docker-compose/blob/master/README.md) file. This project is far from complete and we do not recommend it for production use, only for development and staging systems. MyBB's current installation and upgrade systems make full automation with Docker a difficult feat to acomplish but we're hoping to include easier ways to do this in the future 1.10 series of releases.
+
+We hope that avid plugin and theme developers make good use of the Docker images that we'll be releasing and improving upon, any and all contructive feedback (and pull requests!) is welcome.
+
+Thanks,
```
Comment (dvz) [inline november-2017/docker.md]: We don't use closings in regular Blog posts actually, so that can be dropped

Comment (kawaii) [reply november-2017/docker.md]: I think that offering people to make PRs and giving feedback is never a bad idea, surely?

Comment (dvz) [reply november-2017/docker.md]: Refers to the _Thanks_ part only

Comment (kawaii) [reply november-2017/docker.md]: Ah okay! I'll chop that off now. :slightly_smiling_face:

**Pull request merged**